readme.txt

```diff
@@ -111,30 +111,42 @@ If you get stuck, you can ask for help in the [Plugin Forum](https://wordpress.o
 == Changelog ==
 
 = 9.7.0 - xxxx-xx-xx =
-* Update - Removes BNPL payment methods (Klarna and Affirm) when other official plugins are active
+
+**New Features**
+
+* Adds two new safety filters to the subscriptions detached debug tool: `wc_stripe_detached_subscriptions_maximum_time` and `wc_stripe_detached_subscriptions_maximum_count`
+* Show a notice when editing an active subscription that has no payment method attached
+* Show payment methods sync status on the UI
+* Add the number of pending webhooks to the Account status section
+* New WooCommerce Debug Tool to list subscriptions without a payment method attached
```

Comment:
We should show this feature before the safety filters entry above, as those filters only make sense in the context of the new tool.